src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,117 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(nullable = false)
```

Comment:
We recommend using `@NotNull` (`javax.validation.constraints.NotNull`) instead of `@Column(nullable=false)`. That way the nullcheck is made before going to the database.